src/App.tsx

```diff
@@ -17,18 +73,18 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter setStatusFilter={setStatusFilter} statusFilter={statusFilter} setTitleFilter={setTitleFilter} titleFilter={titleFilter} />
```

Comment:
```suggestion
              <TodoFilter 
                setStatusFilter={setStatusFilter} 
                statusFilter={statusFilter} 
                setTitleFilter={setTitleFilter} 
                titleFilter={titleFilter} 
              />
```

format it correctly. make it readable